build.sbt

```diff
@@ -42,7 +46,7 @@ val installAll =
      |&& gem install activesupport
      |&& gem install parser:2.4.0.0
      |&& gem install pry
-     |&& gem install rubocop:${rubocopVersion}
+     |&& gem install rubocop:$rubocopVersion
```

Comment:
Still, I'm not sure, but probably if you just run `gem install rubocop` it will install the latest rubocop version? Maybe that's the safest default value. This means my solution would be something like:

`gem install rubocop${rubocopVersion.map(version => s":$version").getOrElse("")}` (this probably doesn't work as it's written because of quotes and string interpolation chars nested)